cgp/genome.py

```diff
@@ -346,6 +349,27 @@ def _is_active_input_gene(self, gene_idx: int) -> bool:
         else:
             assert False  # should never be reached
 
+    def _select_gene_indices_for_mutation(
+        self, mutation_rate: float, rng: np.random.RandomState
+    ) -> List[int]:
+        """Selects the gene indices for mutations
+
+        Parameters
+        ----------
+        mutation_rate : float
+            Proportion of genes to be mutated, between 0 (excluded) and 1 (included)
```

Comment:
```suggestion
            Probability of a gene to be mutated, between 0 (excluded) and 1 (included)
```
please also fix this in the docstring of  `Population.__init__`

Reply:
?